src/node_crypto.cc

```diff
@@ -518,6 +518,12 @@ void SecureContext::Init(const FunctionCallbackInfo<Value>& args) {
                                  SSL_SESS_CACHE_NO_AUTO_CLEAR);
 
   SSL_CTX_set_min_proto_version(sc->ctx_.get(), min_version);
+
+  if (max_version == 0) {
+    // Selecting some secureProtocol methods allows the TLS version to be "any
+    // supported", but we don't support TLSv1.3, even if OpenSSL does.
+    max_version = TLS1_2_VERSION;
```

Comment:
Does this need to be documented anywhere?

Reply:
No, its the current behaviour. Right now, `0` for max means `TLS1_2_VERSION`, but if someone   compiles (or dynamically links) against openssl 1.1.1 it starts to means `TLS1_3_VERSION`. With this change, it goes back to meaning what it used to, so no docs required.